packages/react/src/index.tsx

```diff
@@ -11,3 +11,4 @@ export * as primitives from './primitives';
  * Re-export public APIs from `@aws-amplify/ui`
  */
 export { defaultTheme, createTheme, translations } from '@aws-amplify/ui';
+export type { Theme } from '@aws-amplify/ui';
```

Comment:
We were previously exporting this type from within the AmplifyProvider with some weird renaming stuff, this is much cleaner.